pkg/static/login.js

```diff
@@ -888,7 +888,11 @@
                     }
                 }
             } else if (xhr.status == 403) {
-                login_failure(decodeURIComponent(xhr.statusText) || _("Permission denied"));
+                if (!xhr.statusText || xhr.statusText.indexOf("Permission denied") > -1) {
```

Comment:
We could just wrap it into `_()`. In this case `Permission denied` is known string so it would work equally and it might also be able to translate other strings.

```
                login_failure(_(decodeURIComponent(xhr.statusText)) || _("Permission denied"));
```

Reply:
Updated accordingly.